docs/api_reference/src/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+cmake_minimum_required(VERSION 3.24.0)
+project(mp-units_reference_documentations LANGUAGES NONE)
+
+include(JEGPAddStandardeseSources)
+
+set(pdf_title "mp-units Library")
+set(page_license "MIT License")
+set(first_library_chapter "qties")
+set(last_library_chapter "qties")
+set(cover_title "mp-units Library Reference Documentations")
+set(reply_to "\\href{${PROJECT_HOMEPAGE_URL}/discussions}{Discussions}, \\href{${PROJECT_HOMEPAGE_URL}/issues}{issues}")
+jegp_add_standardese_sources(
+    mp-units_reference_documentations
+    LIBRARIES intro quantities
+    EXTENSIONS macros_extensions
+    CHECKED TRUE
+#    PDF PATH "mp-units.pdf" #[[EXCLUDE_FROM_MAIN]]
```

Comment:
It's important to be able to build the PDF so that any LaTeX error can surface.
Were you also unable to build the PDF in CI?
Perhaps <https://github.com/cplusplus/draft>'s CI setup could serve as a reference.

Reply:
Doesn't the HTML version also process all the LaTeX files? I thought I would save time by skipping the PDF generation.

Reply:
Yes, but it isn't a LaTeX interpreter, so there might be errors.
The sources `cxxdraft-htmlgen` processes from <https://github.com/cplusplus/draft> are CI green.

Reply:
I sent you a DM on Discord with the logs.